pkg/workflow_execution.go

```diff
@@ -379,13 +379,16 @@ func (c *Client) CreateWorkflowExecution(namespace string, workflow *WorkflowExe
 	}
 
 	appendSysUid := true
-	for key, _ := range *opts.Labels {
-		if key == "sys-uid" {
+	for _, param := range opts.Parameters {
+		if param.Name == "sys-uid" {
 			appendSysUid = false
 		}
```

Comment:
Add a `break` here since there is no need to continue the loop once the name if found.